### ISSUE TITLE
Phantom console msg: Unable to post message to http://http. Recipient has origin http://http://111.93.41.194:8888.

### ISSUE THREAD

**Issue by dsamit** [ajam/banquo#5]
Hi,
I am Having issue with connecting a webpage and taking screenshot of selected target element.

I am using below options for connection.

var opts = {
    mode: 'save',
    url: '111.93.41.194:8888/?eid=593&mid=408&uid=MTU=&sid=75dk5mau78qkcpjacio2fqhpv2',
    viewport_width: 1440,
    delay: 10000,
    selector: '#articleHighlightList-0',
    scrape: false,
    out_file:'export files/new.png'
};

This is what the issue i am getting..
![vdfha](https://cloud.githubusercontent.com/assets/12273993/9718674/863fd3ca-559b-11e5-8816-309f00607ea4.png)

Please help me out.

**Comment by abenrob** [comment]
When I try to view the url, I get a modal telling me the session is not found, and am then redirected to http://ds09.projectstatus.co.uk/ideascast/

**Comment by mhkeller** [comment]
This looks like an issue with the web page you're trying to access as opposed to phantom or banquo. @dsamit can you give us any more details on what you're trying to capture.

**Comment by dsamit** [comment]
This May be an issue with the web page.
Yes, this web page checks for session availability and the above URL will not have session.  

But the problem is the URL which i tried to that has session but then also it was not able to take screenshot.

If you can give a specific timing when you can be available then i can give you the working URL with valid session. With this it will be easy for you to track down the issue.

**Comment by abenrob** [comment]
@dsamit Just noticing you are using the selector that is defined in the example: '#articleHighlightList-0'.
Im guessing your page does not have an element with the same ID as that; perhaps that is the issue?

**Comment by dsamit** [comment]
Yes. Usually i didn't have this selector in my page. But when this was not working, i thought to change my selector id with articleHighlightList-0 and hence i did this.
But then also it was unable to take screenshot of my webpage.

**Comment by mhkeller** [comment]
Do you have a url for what you want to screenshot that is stable — that is to say, it doesn't change based on a session id or cookie. If so, paste it here and we can see the next step for you.
